### DISCUSSION TITLE
Copier/cruft integration

### DISCUSSION THREAD

**Discussion by jackmac92** [renovatebot/renovate#32760] [Suggest an Idea]
### Tell us more.

related to #24000

[copier](https://github.com/copier-org/copier) is very similar to cruft, which is why the issue is related.

mainly curious about https://github.com/renovatebot/renovate/discussions/24000#discussioncomment-6806128

Is there some obstacle/inherent conflict with how templated repos would update relative to the ususal dep list/lockfiles (this question is targeted at the issues with cruft integration, but the same should apply to copier integration)

**Comment by rarkins** [comment]
Copier already has support: https://docs.renovatebot.com/modules/manager/copier/

**Comment by jackmac92** [comment]
gotcha sorry for not checking that
